basis/common-dao/src/main/resources/changelog/changelog.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="Alex" id="changelog-1.0">
+        <createTable tableName="users">
+            <column name="id" type="VARCHAR(36)">
+                <constraints nullable="false" primaryKey="true" unique="true"/>
+            </column>
+            <column name="phoneNumber" type="TEXT">
+                <constraints nullable="false" primaryKey="false" unique="false"/>
+            </column>
+        </createTable>
+        <createTable tableName="devices">
+            <column name="id" type="VARCHAR(36)">
+                <constraints nullable="false" primaryKey="true" unique="true"/>
+            </column>
+            <column name="userId" type="TEXT">
+                <constraints nullable="false" primaryKey="false" unique="false"/>
```

Comment:
@Alexey007gold Why you need primaryKey="false" unique="false" ? It's default values.